### PR TITLE
Added work and about section

### PR DESCRIPTION
**Pull request by GraceHtet**
Added work and about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,16 +21,178 @@
...
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
```
Comment (DuaneDave) [inline index.html]: In order to have a flexible layout (which you would do in future milestones), it is required to align the contents inside each cards with flex.

File: index.html [hunk 2]
```diff
@@ -18,16 +21,178 @@
...
         </div>
     </section>
-    <!-- END HEADLINE SECTION -->
+    <!-- End HEADLINE SECTION -->
+
+    <!-- Start WORK SECTION -->
+    <section class="works">
+        <!-- start title -->
+        <div class="title">
+            <h1>Projects</h1>
```
Comment (DuaneDave) [inline index.html]: In order to have a well ranking of content, it is advisable to have just one `h1` throughout a website which you have done right at the headline section. Every other opening heading should at least be an `h2`. This also makes a website easily navigatable and readable.

I recommend using an html `h2` tag here.

File: index.html [hunk 3]
```diff
@@ -18,16 +21,178 @@
...
+
+        <!-- start project card -->
+        <div class="projects">
+            <!-- start card1 -->
+            <div class="project-cards project1">
+                <div class="pjimgs">
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
```
Comment (DuaneDave) [inline index.html]: Having a proper ranking of contents in mind, it is advisable that all occurrences of the opening heading for your cards should be a heading tag (at least an `h3` tag as opposed to using a `p` tag).

File: index.html [hunk 4]
```diff
@@ -18,16 +21,178 @@
...
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
```
Comment (DuaneDave) [inline index.html]: As required in the project all occurrences of a button should have multiple states (hover, active, and so on) to make them visually appealing.
I suggest you style these states as given in the figma template, see below

btn states

Comment (DuaneDave) [inline index.html]: All occurrences of a button in each card should at least look like the design from figma. Currently your buttons do not have the rightward pointing arrow as the template. 

Your copy 👇👇

no arrow

Template design 👇👇

with arrow


Kindly implement this according to the design and also take care of the wrapping text by reducing the font size.

File: index.html [hunk 5]
```diff
@@ -18,16 +21,178 @@
...
+        <!-- end project card -->
+    </section>
+    <!-- End WORK SECTION -->
+
+    <!-- Start ABOUT SECTION -->
+    <section class="about">
+        <div class="aboutintro">
+            <!-- start title -->
+            <div class="title">
+                <h1>About</h1>
```
Comment (DuaneDave) [inline index.html]: As explained before, kindly update this to an `h2` tag.

File: index.html [hunk 6]
```diff
@@ -18,16 +21,178 @@
...
+            </div>
+            <!-- end title -->
+            <div class="bio">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+
+            <button type="button" class="cvbtn"><a href="#"><span>Get My CV</span><img src="./img/downloadicon.svg" alt="icon"/></a></button>
```
Comment (DuaneDave) [inline index.html]: One key feature of a great-looking website is that they utilize proper spacing between contents, as such I recommend adding proper spacing between this button and the content which follows.

Your current copy 👇👇👇

no margin




Template design 👇👇👇


margin


Kindly add some `margin-bottom` property here.

File: index.html [hunk 7]
```diff
@@ -18,16 +21,178 @@
...
+        <div class="section-card">
+            <ul class="card-list">
+                <li class="cards card1">
+                    Languages
+                    <p>
+                        • JavaScript (ES5/ES6)<br/>
+                        • Ruby<br/>
+                        • HTML<br/>
+                        • CSS
+                    </p>
```
Comment (DuaneDave) [inline index.html]: Please make this item a list item inside the a `ul` tag to make your code more semantic

File: index.html [hunk 8]
```diff
@@ -18,16 +21,178 @@
...
+                </li>
+                <li class="cards card2">
+                    Framework
+                    <p>
+                        • Bootstrap<br/>
+                        • Ruby  on rails<br/>
+                        • RSpec<br/>
+                        • Capibara<br/>
+                        • Selenium
+                    </p>
```
Comment (DuaneDave) [inline index.html]: As required in the project requirements, all contents in these cards are supposed to be aligned with `flex` as opposed to a `position` property to ensure consistency of layout on all media break points.

Please do this for others as well.

File: style.css [hunk 9]
```diff
@@ -92,11 +96,199 @@ body{
...
+    font-family: 'Playfair Display', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 55px;
+    line-height: 100%;
+    margin:0 0 35px 20px;
+}
+.project-cards{
+    background-color:var(--primary-color);
+    display:flex;
```
Comment (DuaneDave) [inline style.css]: I suggest creating a separate class that makes use of this `display: flex` property so you can truly reuse it without always repeating this rule again.

Like this...
```

.flex {
  display: flex;
}
```

You can then apply this as needed in the container as a class.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @GraceHtet 👋, 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Outstanding design 💯 
✔️ Nice use of HTML tags 🥇 
✔️ Linters are pasing 💯 


## Required Changes ♻️

- Kindly add a more descriptive PR summary to make it it easier to tell what you have accomplished in this milestone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly remove unused sections from your read me file, and also add contents to some required part (eg FAQ, install and so on)
- I noticed thatt the required background is not  in your headline section, please fix this. See  below.

Your design 👇👇👇

UI



Template design 👇👇👇

figma UI
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (GraceHtet) [reply index.html]: The project btn style isn't included in one of those three style what should I do for those btns?  Should I just Leave like this?@DuaneDave 
Screenshot 2022-12-02 205208

File: index.html [hunk 10]
```diff
@@ -18,16 +21,178 @@
...
+                    <img src="./img/pj1.svg" alt="pj1"/>
+                </div>
+                <div class="pjcontent">
+                    <h3 class="pjname">Project name goes here</h3>
+                    <ul >
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
```
Comment (DuaneDave) [inline index.html]: Good job adding flex to this card but I have seen that you didn't add it for the other cards as well, please implement it for all other cards in this section.

Comment (DuaneDave) [inline index.html]: As requested earlier, kindly add the required styling for each button state and make the button design match the one in the figma template.

HINT:  You can get the arrow icon from [here](https://boxicons.com/)

File: index.html [hunk 11]
```diff
@@ -18,16 +21,178 @@
...
+            <!-- start title -->
+            <div class="title">
+                <h2>About</h2>
+            </div>
+            <!-- end title -->
+            <div class="bio">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+
+            <button type="button" class="cvbtn"><a href="#" class="flex"><span>Get My CV</span><img src="./img/downloadicon.svg" alt="icon"/></a></button>
```
Comment (DuaneDave) [inline index.html:159]: As previously requested, kindly align this button and add the required spacing.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @GraceHtet  👋 , 

Great work on making some of the previously requested changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Quick implementation of previously requested changes 👍
✔️ Meaningful commit history 👍 
✔️ Great UI  💯 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 12]
```diff
@@ -84,31 +96,204 @@ body{
...
+.pjbtn{
+    width:60%;
+    background: var(--primary-color);
+    font-size: 16px;
+    font-family: 'Poppins';
+    font-weight: 300;
+    border: 1px solid #FFFFFF;
+    backdrop-filter: blur(0.5px);
+    padding:10px;
+}
```
Comment (Gopxfs) [inline style.css:190]: As requested by the previous reviewer, please make sure to add the corresponding effect to these buttons:
image

You should use these effects:
image

See that the background is dark and the letter is white? It is the same styling 😄!

Tip for this button:
image

You can add a:
```css
border: 0;
```
To make it fit into the original design 😉.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @GraceHtet 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @GraceHtet >, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**